pkg/internalapis/cnsvolumeoperationrequest/v1alpha1/cnsvolumeoperationrequest_types.go

```diff
@@ -46,6 +46,12 @@ type CnsVolumeOperationRequestStatus struct {
 	// storage quota during a create/expand volume operation.
 	// +optional
 	Reserved *resource.Quantity `json:"reserved,omitempty"`
+	// ID of the storage policy
+	// +optional
+	StoragePolicyId string `json:"storagepolicyid,omitempty"`
+	// Name of the storage policy
+	// +optional
+	StoragePolicyName string `json:"storagepolicyname,omitempty"`
```

Comment:
do we need storage policy name as well ?

Reply:
update CRD instance for this API as well

Reply:
Done